### PR TITLE
Record and print error statements

### PR DESCRIPTION
**Pull request by rautenrieth-da**
This PR changes OracleAround such that it records all SQL statements that produce errors, and prints them when the Oracle user is deleted.

### CODE REVIEW

File: libs-scala/oracle-testing/src/main/scala/testing/oracle/OracleAround.scala [hunk 1]
```diff
@@ -76,22 +86,76 @@ object OracleAround {
...
+            l_sql_text ora_name_list_t;
+            l_n        number;
+            v_stmt     clob;
+        begin
+            l_n := ora_sql_txt( l_sql_text );
+            for i in 1 .. l_n loop
+                v_stmt := v_stmt || l_sql_text(i);
+            end loop;
+
+            insert into $serverErrorsTable values ( ora_login_user, ora_server_error(1), v_stmt );
```
Comment (nmarton-da) [inline libs-scala/oracle-testing/src/main/scala/testing/oracle/OracleAround.scala:124]: not sure what this is doing: what is the additional information what we see here besides in the exceptions?
Is it only the SQL statement itself?

**Review by nmarton-da — COMMENTED** [review]
Interesting approach, not sure about the direction itself:
* We might want to improve not only for tests, but also for production / deployed use cases as well
* We might want to improve for all supported DB backends at the same time

For example I can easily imagine some additional support for our anorm String interpolation code to report the constructed SQL in question, if something bad happens.

File: libs-scala/oracle-testing/src/main/scala/testing/oracle/OracleAround.scala (hunk 1 above)
Comment (rautenrieth-da) [reply libs-scala/oracle-testing/src/main/scala/testing/oracle/OracleAround.scala:124]: This was meant as a development time tool to simplify debugging. It only prints the SQL statement, as it was seen by the Oracle server. I was annoyed by the fact how hard it is to see the result of anorm string interpolation (for example, the `#$` operator is applied in anorm code and not our code).

I would not want to deploy this in production. Database operators have other tools to audit their Oracle database.

This is also not needed for Postgres, as Postgres includes the SQL statement in the exception message, and even points out where the error is.

**Comment by stefanobaghino-da** [comment]
Stale?

**Pull request closed without merging**